sunpy/map/mapbase.py

```diff
@@ -3132,6 +3133,11 @@ def reproject_to(self, target_wcs, *, algorithm='interpolation', return_footprin
             target WCS, the extent will be shifted/expanded/cropped by an integer number
             of pixels.
             Defaults to ``None``.
+        preserve_date : `bool`
+            If ``True``, ``'DATE-OBS'`` of the original map will be preserved in the
+            header of the reprojected map. Note that this will not affect the obstime of
+            the reprojected map as this coordinate frame is defined by ``'DATE-AVG'`` which
+            will be the obstime of the target WCS.
```

Comment:
```suggestion
            If ``True``, the observation time of the reprojected map is set to the
            observation time of the original map instead of the observation time defined in the
            target WCS.  This does not affect the reprojection itself, which instead depends on
            the reference times of the two coordinate frames.
            Defaults to ``False``.
```
Your description is too jargon-y, incorrect in some cases, and very easy to misinterpret even in the cases that it is correct

Reply:
FWIW, based on previous reviews, I think you and @Cadair have different opinions about the level of "jargon" to be included in this docstring. Fight amongst each other if you wish.